parallaxinc/Parallax-IDE
# new version overlay

#### What's this PR do?

Adds an overlay when a new version or new installation of the app occurs.
#### Where should the reviewer start?

start with client.js to see the lifecycle of the check, and background.js to see how the app is flagged as updated.
#### How should this be manually tested?

build and reload the app, when launching it should show the overlay.  If you just close and re-open without reload or re-installing the overlay should not appear.
#### What are the relevant tickets?

fixes #215
